content/references/data-api.md

```diff
@@ -2994,19 +2962,19 @@ A successful response uses the HTTP code **200 OK** and has a JSON body with the
 |:-------------|:-----------------------------------------|:-------------------|
 | `result`     | String                                   | The value `success` indicates that the body represents a successful response. |
 | `count`      | Integer                                  | Number of validators returned. |
-| `validators` | Array of Single Validator Report Objects | Daily reports of this validator's validation votes. |
+| `reports`    | Array of Single Validator Report Objects | Daily reports of this validator's validation votes. |
 
-Each member in the `validators` array is a Single Validator Report Object with data on that validator's performance on that day. Each has the following fields:
+Each member in the `reports` array is a Single Validator Report Object with data on that validator's performance on that day. Each has the following fields:
```

Comment:
Consider adding a `#### Single Validator Report Object` or `#### Report Object` heading above this paragraph, just to match style/structure used for other methods?